rules/typescript.js

```diff
@@ -250,9 +250,9 @@ module.exports = {
     '@typescript-eslint/prefer-nullish-coalescing': [
       'error',
       {
-        forceSuggestionFixer: false,
         ignoreConditionalTests: true,
         ignoreMixedLogicalExpressions: true,
+        ignoreTernaryTests: false,
```

Comment:
This is a new option for this rule, so I thought worth adding here. To me `false` makes more sense, but let me know if you think this should be configured as `true`, here's [more details](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md#ignoreternarytests) on how it works.

Reply:
`false` insures consistency and, personally, readability.  `??` is much easier to visually parse in PR than comparisons to undefined and/or null.